atmo/clusters/views.py

```diff
@@ -11,14 +11,15 @@
 from ..decorators import (delete_permission_required, modified_date,
                           view_permission_required)
 from .forms import NewClusterForm
-from .models import Cluster
+from .models import Cluster, EMRRelease
 
 
 @login_required
 def new_cluster(request):
     initial = {
-        'identifier': '{}-telemetry-analysis'.format(user_display(request.user)),
+        'identifier': '%s-telemetry-analysis' % user_display(request.user),
```

Comment:
Hmyeah not really, the old format wasn't removed from Python 3, so there is no reason to use it, especially since it has some spotty support in things like translations, so it's easier to just stick with one formatting. Plus there was [this thing recently](http://lucumr.pocoo.org/2016/12/29/careful-with-str-format/) which made me just go back again.